### PR TITLE
Adding a file about Los Angeles

### LINKED ISSUE #56: Create a new file for api1234 hometown
## Add file for Los Angeles
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request
- [x] Merge our pull request

### PR DESCRIPTION
**Pull request by apoorva89**
Which happens to be @api1234's hometown

This pull Request closes #56

### CODE REVIEW

File: places/los-angeles.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Los Angeles, CA
+
+### Great Places to Eat
```
Comment (apoorva89) [inline places/los-angeles.md:3]: @api1234 I :heart: Continental

**Comment by woodd00** [comment]
Lots of choices in LA I bet.

**Comment by woodd00** [comment]
Looks good!

**Comment by apoorva89** [comment]
@woodd00 Thanks!

**Pull request merged**